package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "open-map-kit-server",
-  "version": "0.6.1",
+  "version": "0.7.0",
```

Comment:
`npm version minor` is a really nice way to do this, as it creates a commit and tag at the same time (and always does it the same way).